src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/generated.py

```diff
@@ -126,3 +129,55 @@
 cli_storage_data_plane_command('storage file service-properties update', FileService.set_file_service_properties, factory)
 cli_storage_data_plane_command('storage file copy start', FileService.copy_file, factory)
 cli_storage_data_plane_command('storage file copy cancel', FileService.abort_copy_file, factory)
+
+# table commands
+factory = table_data_service_factory
+cli_storage_data_plane_command('storage table generate-sas', TableService.generate_table_shared_access_signature, factory)
+cli_storage_data_plane_command('storage table stats', TableService.get_table_service_stats, factory)
+cli_storage_data_plane_command('storage table service-properties show', TableService.get_table_service_properties, factory)
+cli_storage_data_plane_command('storage table service-properties update', TableService.set_table_service_properties, factory)
+cli_storage_data_plane_command('storage table list', TableService.list_tables, factory, simple_output_query='items[*].{Name:name}')
```

Comment:
Added this to all the places it was missing.

---
In reply to: [74452574](https://github.com/Azure/azure-cli/pull/681#discussion_r74452574) [](ancestors = 74452574)